### DISCUSSION TITLE
[BP#05] Favoriser un design simple, épuré, adapté au web

### DISCUSSION THREAD

**Discussion by yannkozon** [cnumr/best-practices#104] [Bonnes pratiques] [answered]
Le terme design porte normalement plutôt sur la conception et non l'interface, la règle présente dans la bonne pratique parle pourtant de l'interface.
J'aurais tendance à reformuler : "Favoriser une conception simple, épurée et adapté au support numérique" pour parler uniquement de la conception qui aura un impact sur l'interface et ne pas limiter ça au web mais au support numérique.

**Comment by ACTLEM** [comment]
Comme nous sommes sur de l'écoconception de services numérique à architecture web, garder le terme "adapté au web" me semble opportun. Par contre, je suis pour modifier le terme "design" à remplacer par "interface" pour éviter une confusion entre le titre et la description.

**Reply by tbroyer** [reply]
"interface utilisateur" alors, pour ne pas confondre avec API :wink:

**Comment by florinesueur** [comment]
Je me demande pourquoi ne pas parler d'ergonomie ou plus précisément d'heuristiques plutôt que de simple et épuré qui reste à l'appréciation de chacun, il me semble. 

Par exemple, les heuristiques de [Scapin et Bastien](https://blocnotes.iergo.fr/concevoir/les-criteres-heuristiques-de-bastien-et-scapin/).

**Comment by ACTLEM** [comment]
Côté équipe projet, la majorité est plutôt pour garder le titre. Par contre, nous ne sommes pas du tout contre pour préciser dans la description. @florinesueur @yannkozon vous seriez ok pour retravailler un peu le contenu afin de l'améliorer?

**Reply by florinesueur** [reply]
Ok pour travailler sur le contenu de la description.

**Comment by ACTLEM** [comment, marked as answer]
J'ai créé l'issue pour améliorer le contenu (interface utilisateur, heuristique...): https://github.com/cnumr/best-practices/issues/251
